### PR TITLE
iputils: update to 20240117

### PR DESCRIPTION
**Pull request by krant**
Maintainer: @nmeyerhans 
Run tested: ARMv7, Linksys WRT3200ACM, master branch

Description:
- Rebase the patch
- Don't set Meson options which are matching defaults

### CODE REVIEW

File: net/iputils/Makefile [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
-PKG_RELEASE:=2
+PKG_VERSION:=20240117
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/iputils/iputils/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=b6f67fc705490673ff4471d006221b4a2f1b1180b929d9fefd771352621ccedf
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/iputils/iputils/releases/download/$(PKG_VERSION)
+PKG_HASH:=2c3f1f4d6f2481dec05b441de9c81d4059bdb202d4d0e98404f62ee1502bf1dc
```
Comment (BKPepe) [inline net/iputils/Makefile:17]: Why this change?

**Comment by neheb** [comment]
I merged the other PR.
